src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -15,7 +15,20 @@
  */
 public class Intake extends SubsystemBase {
 
-    public static final double INTAKE_RATE = 0.01;
+    /**
+     * Enum for Intake States.
+     */
+    public enum States {
```

Comment:
this should never be in an unknown state so it should be removed beacuse we always know the state of the pneumatics